data/human/fleets.txt

```diff
@@ -8,6 +8,32 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+fleet "Southern Patrol Drones"
```

Comment:
I'm not a fan of these fleets. Not only does this create scanners in southern space to catch a player's illegal cargo where normally there were not, but these also basically have the same purpose as the Flycatcher, which gets into the realm of using mechanics/ideas in the early game instead of saving them for later to make exploration more interesting (talked about in #4986). These fleets also run into the issue of systems that the player stays in with these fleets becoming filled to the brim with drones.